js/CourseParser.js

```diff
@@ -1,81 +1,81 @@
-function CourseParser() {
```

Comment:
I'm not sure why the diff shows this area as all deleted and replaced. I checked the website on my machine and all the regular expressions and Javascript seemed to work fine.